lib/spack/spack/installer.py

```diff
@@ -575,6 +575,8 @@ def package_id(pkg):
             keep_stage (bool): By default, stage is destroyed only if there
                 are no exceptions during build. Set to True to keep the stage
                 even with exceptions.
+            overwrite (list): list of hashes for packages to do overwrite
```

Comment:
I think the `force` option was previously used for overwrites, but is no longer respected (and overwrites were handled in the install command instead of the installer). I'll remove it.